runtime/port/unix/j9sysinfo.c

```diff
@@ -838,24 +838,33 @@ getS390Description(struct J9PortLibrary *portLibrary, J9ProcessorDesc *desc)
 	if (J9_ARE_NO_BITS_SET(*(int*) 200, S390_STFLE_BIT)) {
 		return -1;
 	}
-#elif defined(LINUX) && !defined(J9ZTPF) /* LINUX S390*/
+#elif defined(J9ZTPF)  /* ZTPF */
```

Comment:
It's a nitpick but the `#if / #elif / #else` close comments need to be adjusted to indicate the end of the previous one.

`#elif defined(J9ZTPF)  /* J9ZOS390 */`
and

`#elif defined(LINUX) /* J9ZTPF */`

Reply:
Thanks, done.